packages/cli/src/commands/generate/function/templates/function.ts.template

```diff
@@ -0,0 +1,10 @@
+import { APIGatewayEvent, Context } from 'aws-lambda'
```

Comment:
```suggestion
import type { APIGatewayEvent, Context } from 'aws-lambda'
```

Reply:
This'll be cool cause I did not know about this functionality and it'll be provided for all TS-functions.